projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -18,7 +18,49 @@ import React from 'react';
  */
 import styles from './styles.module.scss';
 
+const getConnectionsDiv = () => {
```

Comment:
Rather than a function, you could also turn this into a component (which really is just a function anyway). I'd also recommend not using the name of the HTML element in your function/component name. You could choose something like `<ConnectionItems>`.

`const ConnectionItems = () => {`